pkg/arvo/app/launch.hoon

```diff
@@ -40,36 +41,55 @@
       path-to-tile=(map path @tas)
       first-time=?
   ==
++$  state-two
+  $:  %2
+      tiles=(set tile:launch)
+      data=tile-data:launch
+      path-to-tile=(map path @tas)
+      first-time=?
+      setup-who=_%.y
```

Comment:
Doesn't look like we need to keep this flag around? We don't use it anywhere, and idk what we'd use it for.